### ISSUE TITLE
Websocket failed, polling doesn't work

### ISSUE THREAD

**Issue by CarlSinclair** [Drakkar-Software/OctoBot#2621]
## Step 1: Have you search for this issue before posting it?
Yes

## Step 2: Describe your environment
  * OS : [AlmaLinux 9, docker] 
  * Python Version: 3.29
  * In case you are not using a binary version:
    * Branch: Master
    * Last Commit ID: _____ (`git log --format="%H" -n 1`)
 
## Step 3: Describe the problem:
**Describe the bug**
Watched markets never loads, trading page never loads, and the following errors appear in console:
![image](https://github.com/Drakkar-Software/OctoBot/assets/69092633/98b40ae2-01cb-4b16-8d5c-48b992e2ddda)

**Expected behavior**
No errors

### Steps to reproduce:

  1. install docker container and have it listen on a random port
  2. create Apache reverse proxy for that random port
  3. start the container
  4. Visit Octobot in the browser

**Comment by CarlSinclair** [comment]
Compared to my Windows install:

![image](https://github.com/Drakkar-Software/OctoBot/assets/69092633/46543ae4-cf25-4663-9ad8-82f0a801f192)

**Comment by Herklos** [comment]
Hi,

Are you using the same setup for you Windows install? (with apache as reverse proxy)

**Comment by CarlSinclair** [comment]
No. On Windows I used the binary, and it worked perfectly. Trying to run the binary on linux fails with the following error:
[3745252] Error loading Python lib '/tmp/_MEIcBGdCW/libpython3.10.so.1.0': dlopen: /lib64/libm.so.6: version `GLIBC_2.35' not found (required by /tmp/_MEIcBGdCW/libpython3.10.so.1.0)

My guess is that's likely due to the fact that the system uses python 3.9 and OctoBot requires 3.10. I installed 3.10 and followed [the python/git](https://www.octobot.cloud/en/guides/octobot-installation/install-octobot-with-python-and-git) instructions. Now the errors in the page are gone, but there are errors in the console:
![image](https://github.com/user-attachments/assets/41042663-737f-4a2e-a39e-0a0cfca718d4)

Allowing trackers removed the errors, but the chart still won't load:
![image](https://github.com/user-attachments/assets/479d45e0-d05d-4197-b980-144cc3c4e02c)

**Comment by CarlSinclair** [comment]
I found the following errors in the notifications:
![image](https://github.com/user-attachments/assets/9dcd2824-c0c5-4329-9545-23dc59901ae3)

Since that URL loads correctly in the browser, I figured it could be a CSP issue. So I added the URL to `default-src` since I have no idea what specific permissions are required. No new errors are generating in the notifications, but the chart still isn't loading. Other pages are stuck on `OctoBot is loading`.

Could this be because I have no exchanges configured? I don't have any configured on the Windows setup either, but all the pages are loading correctly despite the errors in the notifications:
![image](https://github.com/user-attachments/assets/c7675e0d-a2ac-4726-8f35-753e6500a73b)

**Comment by CarlSinclair** [comment]
But like I said, the Windows binary works without adding any Binance credentials.

I'm also not interested in using Binance. I want to use a dex like Bisq. Is that possible? Or would it require significant core changes to the code?

**Comment by KhryptorGraphics** [comment]
I have this issue too, but it is intermittent. It happens no matter what platform its on.

**Comment by Herklos** [comment]
> But like I said, the Windows binary works without adding any Binance credentials.
> 
> I'm also not interested in using Binance. I want to use a dex like Bisq. Is that possible? Or would it require significant core changes to the code?

We are using [ccxt](https://github.com/ccxt/ccxt) to support multiple crypto exchanges. If ccxt does not support an exchange like [Bisq](https://github.com/ccxt/ccxt/issues/1134), you should first ask them to support it and then we should be able to integrate it in OctoBot.
To ask for the support of an new exchange in OctoBot you have to create a post at https://feedback.octobot.cloud/open-source.
